api/bases/glance.openstack.org_glances.yaml

```diff
@@ -120,8 +120,6 @@ spec:
                   in /etc/<service> . TODO: -> implement'
                 type: object
               glanceAPIExternal:
-                default:
-                  replicas: 1
```

Comment:
If I understand correctly what you are saying we cannot set the default replicas to 1 unless we set a default for all the other required fields (eg. containerImage), right?

Reply:
The default value of a struct needs to fill every fields of that struct that are marked as Required otherwise the default value cannot be used to construct a struct instance. See my other comment in glance_types.go below